clients/privacy-center/pages/_app.tsx

```diff
@@ -1,6 +1,12 @@
 import type { AppProps } from "next/app";
 import { SessionProvider } from "next-auth/react";
-import { FidesProvider } from "@fidesui/react";
+/*
+ * This import needed to be updated to '@chakra-ui/react' from "@fidesui/react".
+ * Under the hood fidesui is importing from "@chakra-ui/provider" instead "chakra-ui/react".
+ * This causes issues with toasts because it doesn't set up everything required for them.
+ * Solution found here https://github.com/chakra-ui/chakra-ui/issues/5839#issuecomment-1266493711
+ * */
+import { ChakraProvider } from "@chakra-ui/react";
```

Comment:
great find + comment! it looks like that's how we do it in `admin-ui` too

Reply:
Maybe we should update this in `fidesui`? Either that or get rid of the aliasing types altogether. My guess is someone ran into the same problem I did in the `admin-ui` and did exactly what I did.